### ISSUE TITLE
Cross-Check '2 - Linear Regression and Visualization' - 'Kaktuscha'

### ISSUE THREAD

**Issue by Kaktuscha** [rolling-scopes-school/support#189]
**Repository:** https://github.com/Kaktuscha/Linear_regressionn/blob/master/seminar_and_homework.ipynb
**Task:** 
https://github.com/rolling-scopes-school/ml-intro/blob/2021/2_linear_regression/seminar_and_homework.ipynb
**Form:** 
https://rolling-scopes-school.github.io/checklist/
**Marks:**
Screenshot:
![image](https://user-images.githubusercontent.com/43134172/108530628-b1b19700-72cd-11eb-8737-ac49351a5f19.png)

4 students have checked out 5, and marks did not appear in the total score section.

**Comment by JackNightsky** [comment]
Проверив работу у меня получилась оценка в 95 балов. 
Но потом я глянул бал в  Score таблице и там оказался 0 (ноль). Видется мне, что товарищ не проверял чужие работы. За что и получил свой 0. 
**В изменении оценки смысла не вижу**. 

Средний бал вместе с моей из выставленных составляет (98 + 130 + 95 + 118) / 4 =  110.25
Такую оценку он мог бы получить.

TOTAL SCORE: 8 + 0 + 0.5 - 0.5 + 0.5 + 0.5  + 0.5 + 0 = 9.5
Kaktuscha
===
Task 1 (1 point)   || OK
Task 2 (0.5 point) || OK
Task 3 (1.5 points)|| OK
Task 4 (5 points)  || OK
===
score: 8

Task 5 (up to 5 points)
1. _Extract float number from Value field in DataFrame (0.5 points)_
   **BAD** 60K => 60 000 000 неверная конвертация тысяч. (60K == 60 000) || 0 

2. _Сhoose more features that you expect to influence on player Value (at least 10)_
_Plot feature correlation matrix. (0.5 points)_
  **OK** || 0.5 


3. _Drop features that are highly correlated with each other (abs(corr) > 0.9) one by one until no correlated pairs left. Hint: you may reuse code from Task_9 in HW_1 for automatic correlated pairs selection. (1.5 points)_
  **Penalties: -0.5 points if did not remove linearly dependent features before training the model.** || -0.5

4. Split data into train/test with some proportion (0.5 points)
  **OK** || 0.5 
  
5. _Train a model on train dataset, make predictions both for train and test. (0.5 points)_
  **OK** || 0.5
6. _Measure the model quality in terms of MSE in train and test samples, (0.5 points)_
  **OK** || 0.5
7. _Write a short report about the work done. Why did you take these particular features? Can you find a logical explanation for high correlation of some of your features? Are you satisfied with the quality of predictions? etc. (1 point)_
   **NOT OK** Report is out. || 0

**Comment by Kaktuscha** [comment]
I was a bit lost with the rule I did not know that we have a specific time apart from the date, however I have check all other work on the same date. Maybe you will give a mark for this time, as a lesson learned. Now I am checking other tasks before the specific time. @JackNightsky Please.

**Comment by JackNightsky** [comment]
I would put 0. 
After consulting with other activists, given that this is the first homework assignment, we decided to put the mark on which I checked the work - **95**. I hope now you check the work on the very first day after they appear for review.
@Kaktuscha 
--------
Also, your post-deadline review of other work did not affect the other students' scores. Because grades given after the deadline are not counted.  So you framed these students.
